modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/position/align.js

```diff
@@ -0,0 +1,587 @@
+/**
```

Comment:
If I'm reading the folder structure correctly, this is creating a `position` folder to hold a single `align.js` file. I think @wincent wrote something down about this, or maybe it's time we do so I don't have to keep mentioning him when it happens 😂 

Just flatten this out, same as with the `tests/position/align.js` portion.

Reply:
>  I think @wincent wrote something down about this

Yes, on some Senna PRs...

My hot-take on unnecessary nesting:

![mess](https://user-images.githubusercontent.com/7074/105018872-c9cf9580-5a45-11eb-9efa-52b8f28cb83f.gif)

Reply:
> this is creating a position folder to hold a single align.js
Yes that's true. I agree the `position` directory isn't required, I'll change that.

Reply:
That gif made me :rofl: